LearningMissionSimulation/ObjectGenerator.cs

```diff
@@ -148,5 +148,21 @@ public static Subject GenerateSubject()
             return new Subject(subjectType: AttributeGenerator.GetSubjecType(), description: AttributeGenerator.GetUnitDescprition(UnitType.Subject),
                                name: AttributeGenerator.GetUnitName(UnitType.Subject), itemList: ObjectGenerator.GenerateModuleList());
         }
+
+        //public static Resume GenerateResume()
```

Comment:
Remove comments (uncomment), or remove the code.

Reply:
Removed.